cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -28,6 +28,76 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/generate/collector"
 )
 
+const (
+	//nolint:lll
+	examples = `
+  # Using the example 'memcached-operator' and assuming a directory structure
+  # similar to the following exists:
+  $ tree api/ config/
+  api/
+  └── v1alpha1
+      ├── groupversion_info.go
+      ├── memcached_types.go
+      └── zz_generated.deepcopy.go
+  config/
+  ├── bundle
+  │   └── kustomization.yaml
+  ├── crd
+  │   ├── bases
+  │   │   └── cache.my.domain_memcacheds.yaml
+  │   ├── kustomization.yaml
+  │   ├── kustomizeconfig.yaml
+  │   ...
+  ├── default
+  │   ├── kustomization.yaml
+  │   ...
+  ├── manager
+  │   ├── kustomization.yaml
+  │   └── manager.yaml
+  ...
+
+  # Generate bundle files and build your bundle image with this 'make' recipe:
+  $ export USERNAME=<your registry username>
+  $ make bundle-build BUNDLE_IMG=quay.io/$USERNAME/memcached-operator-bundle:v0.0.1
```

Comment:
we have now 2 targets that will do it. IMO we should have only  `make bundle-build`. See: 

```
$  make bundle
/Users/camilamacedo/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate bundle -q --kustomize
/Users/camilamacedo/go/bin/kustomize build config/bundle | operator-sdk generate bundle -q --manifests --version 0.0.1
camilamacedo@Camilas-MacBook-Pro ~/go/src/pr-bundle $  make bundle-build
/Users/camilamacedo/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate bundle -q --kustomize
/Users/camilamacedo/go/bin/kustomize build config/bundle | operator-sdk generate bundle -q --manifests --metadata --overwrite --version 0.0.1
INFO[0005] Building annotations.yaml                    
INFO[0005] Building Dockerfile                          
operator-sdk bundle validate config/bundle
INFO[0000] Found annotations file                        bundle-dir=config/bundle container-tool=docker
INFO[0000] Could not find optional dependencies file     bundle-dir=config/bundle container-tool=docker
INFO[0000] All validation tests have completed successfully 
docker build -f bundle.Dockerfile -t controller-bundle:0.0.1 .
Sending build context to Docker daemon  42.43MB
Step 1/9 : FROM scratch
 ---> 
Step 2/9 : LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 ---> Using cache
 ---> 91093cb32610
Step 3/9 : LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 ---> Using cache
 ---> 26e40a8a5669
Step 4/9 : LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 ---> Using cache
 ---> c44955a2769a
Step 5/9 : LABEL operators.operatorframework.io.bundle.package.v1=pr-bundle
 ---> Using cache
 ---> aa8a3d023e0f
Step 6/9 : LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 ---> Using cache
 ---> 519a771b893d
Step 7/9 : LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
 ---> Using cache
 ---> 396b25fd7c9f
Step 8/9 : COPY config/bundle/manifests /manifests/
 ---> Using cache
 ---> 65f1b9577b20
Step 9/9 : COPY config/bundle/metadata /metadata/
 ---> Using cache
 ---> 77c3fa71a9ee
Successfully built 77c3fa71a9ee
Successfully tagged controller-bundle:0.0.1
```